frontend/src/scenes/groups/GroupsIntroduction.tsx

```diff
@@ -45,17 +45,17 @@ export function GroupsIntroduction({ access }: Props): JSX.Element {
 
 export function GroupIntroductionFooter({ needsUpgrade }: { needsUpgrade: boolean }): JSX.Element {
     return (
-        <div className="text-sm bg-mid rounded p-2" style={{ maxWidth: '15rem' }}>
+        <div className="text-sm bg-mid rounded p-2 max-w-60">
             {needsUpgrade ? (
                 <>
-                    Track usage of groups of users with Group Analytics.{' '}
```

Comment:
Unbreakable spaces are really useful in avoiding widows. We should keep them, but in the explicit `&nbsp;` form